part-1/part-1.md

```diff
@@ -78,67 +79,84 @@ The main entity (which wraps our data) from that library is the _ **Observable**
 
 ## What are Observables?
 
+Short answer: `Observable` is an object that represents a stream of data, to which we can subscribe to receive events/error notifications, and which we can manipulate using special functions called operators.
```

Comment:
suggestion: "...to receive events/error notifications overtime" instead of just "...to receive events/error notifications"

Reply:
nvm, didn't read the "In the end, ..." part which mentions "over time".